app/src/main/res/values/configuration.xml

```diff
@@ -5,4 +5,5 @@
 <resources>
     <!-- We are overriding this value in values-sw600dp -->
 
+    <integer name="channelTileAnimTime">100</integer>
```

Comment:
It feels weird to put this in `configuration.xml`: it was originally added in Focus to distinguish between configurations, specifically "mobile" and "tablet" and we're not doing that here.

In Android, this would conventionally go in an `integers.xml`. That sounds like a good place to put it to me for now: what do you think?

Also, it seems `values*/configuration.xml` are empty: do you mind removing them while you're here?